pkg/controller/argocd/rolebinding.go

```diff
@@ -157,6 +157,15 @@ func (r *ReconcileArgoCD) reconcileClusterRoleBinding(name string, role *v1.Clus
 		roleBinding = newClusterRoleBindingWithname(name, cr)
```

Comment:
This like can be deleted. (I know you didn't add it.)

Reply:
Also, it looks like `reconcileArgoApplier()` is a dead func.

Reply:
> This like can be deleted. (I know you didn't add it.)

I think we need this. 
When we do a GET on clusterRoleBinding and if it doesn't exist, it populates the `roleBinding` object with an empty struct, so we need to reinitialize it.

Reply:
name and cr do not change in the second call `newClusterRoleBindingWithname(name, cr).   `roleBinding` simply get assinged again without anyone touching.  The second call is clearly not needed.  But, if you are not comfortable removing, then let's leave it.

Reply:
Nevermind.  I missed the RoleBinding passed in to the GET.

Reply:
removed `reconcileArgoApplier()` since it wasn't being called anywhere !!